pom.xml

```diff
@@ -1,12 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2019 CDAP
```

Comment:
This should still be Cask Data, Inc right?

Reply:
The script automatically generates this